dotcom-rendering/src/web/experiments/tests/sign-in-gate-mandatory-long-test.ts

```diff
@@ -0,0 +1,244 @@
+import type { ABTest } from '@guardian/ab-core';
+import { getCountryCodeSync } from '../../lib/getCountryCode';
+import { setOrUseParticipations } from '../lib/ab-exclusions';
+
+// Flag to determine whether the canRun function 'setOrUseParticipations' will set a participation (true)
+// or use localstorage participation key to decide canRun result (false)
+const setParticipationsFlag = true;
+
+const EuropeList = [
```

Comment:
Could this be an explicit array of `CountryCode`s?

Reply:
Suppose this works `const EuropeList: (CountryCode | '')[] = `